### PR TITLE
add an endpoint to get admission plugins for the admin

### PR DESCRIPTION
**Pull request by zreigz**
**What this PR does / why we need it**: add an endpoint to get admission plugins for the admin

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

```release-note
add an endpoint to get admission plugins for the admin: GET /api/v1/admin/admission/plugins
```

### CODE REVIEW

File: api/pkg/handler/v1/admin/admission_plugin.go [hunk 1]
```diff
@@ -0,0 +1,39 @@
+package admin
+
+import (
+	"context"
+	"github.com/go-kit/kit/endpoint"
```
Comment (maciaszczykm) [inline api/pkg/handler/v1/admin/admission_plugin.go:6]: Missing a new line.

Comment (zreigz) [reply api/pkg/handler/v1/admin/admission_plugin.go:6]: done

**Pull request merged**